File: soutLink-frontend/src/app/services/models/detailed-list-response.ts
/* tslint:disable */
/* eslint-disable */
import { DetailedProjectResponse } from '../models/detailed-project-response';
import { SupervisorProjectResponse } from '../models/supervisor-project-response';
export interface DetailedListResponse {
  createdDate?: string;
  id?: number;
  listProjects?: Array<DetailedProjectResponse>;
  supervisor?: SupervisorProjectResponse;
}

File: soutLink-backend/src/main/java/com/soutLink/student/dto/DetailedListResponse.java
package com.soutLink.student.dto;
import com.soutLink.supervisor.dto.DetailedProjectResponse;
import lombok.*;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class DetailedListResponse {
    private Long id;
    private List<DetailedProjectResponse> listProjects;
    private SupervisorProjectResponse supervisor;
    private LocalDateTime createdDate;
}
